addons/website_slides_forum/models/slide_channel.py

```diff
@@ -1,14 +1,30 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from odoo import fields, models
+from odoo import fields, models, api, _
+from odoo.addons.http_routing.models.ir_http import slug
 
 
 class Channel(models.Model):
     _inherit = 'slide.channel'
 
     forum_id = fields.Many2one('forum.forum', 'Course Forum')
+    active_posts_count = fields.Integer('Number of active forum posts', compute="_compute_active_posts_count")
 
     _sql_constraints = [
         ('forum_uniq', 'unique (forum_id)', "Only one forum per slide channel!"),
     ]
+
+    def _compute_active_posts_count(self):
```

Comment:
In the other task I added some computed fields on forum... first one to merge (hint: I am the one who r+) will win.